src/allDrugs.jl

```diff
@@ -273,3 +273,46 @@ function blissCellNum(g1s, g2s; T = 96, n = 8)
     @assert(all(combined .>= 0.0))
     return combined
 end
+
+""" temporal combination of two drugs.
+Here we want to plot a simulation, with first 48 hours being treated with lapatinib, and another 48 hours with gem. """
+function temporal_combination(params1, params2, g0, title)
+    t1 = LinRange(0.0, 48.0, 96)
+    t2 = LinRange(48.5, 95.5, 96)
+
+    g1L, g2L = predict(params1, g0, t1, Int(floor(params1[6])), Int(floor(params1[7])), Int(floor(params1[8])), Int(floor(params1[9])))
+    g1G = g2G = zeros(96)
+    nG1 = Int(floor(params2[6]))
+    nG2 = Int(floor(params2[7]))
+    nD1 = Int(floor(params2[8]))
+    nD2 = Int(floor(params2[9]))
+    v = predict2(params2, g1L[end] + g2L[end], t2, nG1, nG2, nD1, nD2,2)
```

Comment:
`g1L[end] + g2L[end]` is not the correct state at t=48.0. This assumes cells are evenly distributed through the linear chain, which isn't the case as soon as you treat with a drug.